x-pack/plugins/apm/public/components/routing/templates/apm_service_template/index.tsx

```diff
@@ -276,7 +276,14 @@ function useTabs({ selectedTab }: { selectedTab: Tab['key'] }) {
         path: { serviceName },
         query,
       }),
-      append: <BetaBadge icon="beaker" />,
+      append: (
+        <EuiBetaBadge
+          label="B"
+          title="Beta"
+          color="hollow"
+          tooltipContent="This feature is currently in beta. If you encounter any bugs or have feedback, please open an issue or visit our discussion forum."
```

Comment:
You probably need to translate all the texts using `i18n.translate`, besides color.